res/dialogue/dsg/characters/eisek/stall.xml

```diff
@@ -0,0 +1,394 @@
+					#IF(eisek.getAffection(pc) >= 15 && !flags.hasFlag(FLAG_dsg_elis_eisek_asked_himself))
+						[#flags.setFlag(FLAG_dsg_elis_eisek_asked_himself, true)]
+						[#flags.incrementSavedLong("elis_eisek_talk_score", 1)]
+					#ENDIF
+				]]></effects>
+			</response>
+			<response>
+				<responseTabIndex><![CDATA[0]]></responseTabIndex>
+				<index><![CDATA[3]]></index>
+				<responseTitle><![CDATA[Business]]></responseTitle>
+				<responseTooltip><![CDATA[Ask Eisek about how his business is doing.]]></responseTooltip>
+				<nextDialogue><![CDATA[dsg_characters_eisek_stall_talk_business]]></nextDialogue>
+				<secondsPassed/>
+				<effects><![CDATA[
+					#IF(!flags.hasFlag(FLAG_dsg_elis_eisek_asked_business))
+						[#flags.setFlag(FLAG_dsg_elis_eisek_asked_business, true)]
+						[#flags.incrementSavedLong("elis_eisek_talk_score", 1)]
+					#ENDIF
+				]]></effects>
+			</response>
+			<response>
+				<availabilityConditional><![CDATA[[#eisek.getAffection(pc) >= 15]]]></availabilityConditional>
+				<responseTabIndex><![CDATA[0]]></responseTabIndex>
+				<index><![CDATA[4]]></index>
+				<responseTitle><![CDATA[Other dragons]]></responseTitle>
+				<responseTooltip><![CDATA[Ask Eisek if he's ever met other dragons]]></responseTooltip>
+				<nextDialogue><![CDATA[dsg_characters_eisek_stall_talk_other_dragons]]></nextDialogue>
+				<secondsPassed/>
+				<effects><![CDATA[
+					#IF(!flags.hasFlag(FLAG_dsg_elis_eisek_asked_other_dragons))
+						[#flags.setFlag(FLAG_dsg_elis_eisek_asked_other_dragons, true)]
+						[#flags.incrementSavedLong("elis_eisek_talk_score", 1)]
+					#ENDIF
+				]]></effects>
+			</response>
+			<response>
+				<availabilityConditional><![CDATA[[#(!pc.hasQuest(QUEST_LINE_SIDE_EISEK_MOB) && !pc.isQuestCompleted(QUEST_LINE_SIDE_EISEK_MOB))]]]></availabilityConditional>
+				<responseTabIndex><![CDATA[0]]></responseTabIndex>
+				<index><![CDATA[5]]></index>
+				<responseTitle><![CDATA[Mob]]></responseTitle>
+				<responseTooltip><![CDATA[Ask Eisek about the mob that was around his stall the first time you met him.]]></responseTooltip>
+				<nextDialogue><![CDATA[dsg_characters_eisek_stall_talk_ask_about_mob]]></nextDialogue>
+				<secondsPassed/>
+				<effects><![CDATA[
+					#IF(!flags.hasFlag(FLAG_dsg_elis_eisek_asked_about_mob))
+						[#flags.setFlag(FLAG_dsg_elis_eisek_asked_about_mob, true)]
+					#ENDIF
+				]]></effects>
+			</response>
+			<response>
+				<availabilityConditional><![CDATA[[#(pc.hasQuest(QUEST_LINE_SIDE_EISEK_MOB) && pc.isQuestProgressLessThan(QUEST_LINE_SIDE_EISEK_MOB, QUEST_EISEK_MOB_QUEST_STAGE_TWO) && !pc.isQuestCompleted(QUEST_LINE_SIDE_EISEK_MOB))]]]></availabilityConditional>
+				<responseTabIndex><![CDATA[0]]></responseTabIndex>
+				<index><![CDATA[5]]></index>
+				<responseTitle><![CDATA[Mob]]></responseTitle>
+				<responseTooltip><![CDATA[Probably best to not bring up this topic back up without some results to show.]]></responseTooltip>
```

Comment:
Should be: Probably best to not bring this topic back up without some results to show.